lib/active_reporting/database_adapters/mysql_adapter.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module ActiveReporting
+  module DatabaseAdapters
+    # Database adapters are here to solve SQL problems in the
+    # most idiomatic way in each database
+    class MysqlAdapter < Base
+      # Generate SQL snippet with DATE_TRUNC
+      # @param [String] interval
+      # @param [String] field
+      # @return [String]
+      def date_trunc(interval, field)
+        clean_sql(
+          <<-SQL
```

Comment:
The "squiggly heredoc" has been in Ruby since 2.3, I recommend using that instead.
https://ruby-doc.org/core-2.3.0/doc/syntax/literals_rdoc.html#label-Here+Documents

Also, I would say I personally am not concerned by multi-line SQL being generated, so I don't think `#clean_sql` is really necessary.